docs/index.yml

```diff
@@ -43,13 +43,13 @@ highlightedContent:
       title: Office Add-ins Beginner's guide
       url: overview/learning-path-beginner.md
     # Card 7
+    - itemType: sample
```

Comment:
Is there a "Try it out" type?

Reply:
unfortunately there isn't. sample seems to be the closest available option